pxteditor/experiments.ts

```diff
@@ -173,13 +173,6 @@ namespace pxt.editor.experiments {
                 name: lf("Blocks Error List"),
                 description: lf("Show an error list panel for Blocks")
             },
-            {
```

Comment:
> This will make it so if we do a release without enabling across the board, identity will not be enable-able in the live site; is that intentional? I suppose we have a bit of time until then / can worry about it later

This was intentional. I don't think we want identity to be an experiment available in production.